monitor/internal/pod/delete_controller.go

```diff
@@ -31,10 +34,19 @@ type DeleteController struct {
 	reconcileFunc      deleteControllerReconcileFunc
 	tickerPeriod       time.Duration
 	itemProcessTimeout time.Duration
+	sandboxExtractor   extractors.PodSandboxExtractor
+	eventsCh           chan event.GenericEvent
+}
+
+// DeleteObject is the obj used to store in the event map.
+type DeleteObject struct {
+	nativeID  string
```

Comment:
can we rename nativeID to podUID?

we've been changing forth and back here so much ... and my gut tells me that we might not even be at the end of it ... so I'd like to be explicit (even though this is kind of nitpicking ;) )